ZenPacks/zenoss/HttpMonitor/http.py

```diff
@@ -164,13 +168,42 @@ def _getBody(self, response):
         self._response = response
         return readBody(response).addErrback(self._pageErr)
 
-    def _answer(self, body):
+    def _answer(self, body=""):
         res = dict()
-        body = body if not self._body else self._body
+        body = str(body if not self._body else self._body)
         res['body'] = body
         res['headers'] = self._response.headers
         res['code'] = self._response.code
         res['message'] = self._response.phrase
         res['time'] = time.time() - self._startTime
-        res['size'] = self._bodysize(body)
+        res['size'] = self._bodysize(body+str(res['headers']))
```

Comment:
have you verified the result of this against other accepted tools (like the check-http you are replacing)?  It seems to me that the sys.getsizeof function that your _bodysize method uses will return the number of bytes of memory the object is consuming, which might well be different from the actual length of the body of the http response.  Would the response.length be more appropriate?

Reply:
`response.length` doesn't include `headers` length which should be.

ZP Results:
(Pdb) self._bodysize(body+str(res['headers']))
714

(Pdb) self._response.length
534

Nagios Plugin result:
size=703B

Reply:
somehow I saw the response to my comment above in slack, but not in the PR.
If your plugin is returning a different size result than the plugin you are trying to replace, then you should either have a very good explanation of why it is different, or you should fix your plugin to return the same size result.  Perhaps response.length + len(str(res['headers']))?  what do you need to do to make it match?